fs/sync/sync_test.go

```diff
@@ -154,7 +154,7 @@ func testCopyMetadata(t *testing.T, createEmptySrcDirs bool) {
 	if features.ReadDirMetadata {
 		fstest.CheckEntryMetadata(ctx, t, r.Fremote, fstest.NewDirectory(ctx, t, r.Fremote, dirPath), dirMetadata)
 	}
-	if !createEmptySrcDirs {
+	if !createEmptySrcDirs || !features.CanHaveEmptyDirectories {
```

Comment:
I fixed this one in 76798d5bb1382ffd191a0b685178eee0751bf3b4 - I'd forgotten I hadn't merged your fix - sorry!